vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -2472,6 +2489,12 @@ void vtkF3DRenderer::ConfigureRangeAndCTFForColoring(
     return;
   }
 
+  // Set Discretization
+  if(!this->DiscretizableColorTransferFunctionConfigured) {
+    this->DiscretizableColorTransferFunction->SetNumberOfValues(this->Discretization);
```

Comment:
Can you do that directly on the existing colorTransferFunction

Reply:
I think you misunderstood, you can use SetNumberOfValues on the existing colorTransferFunction ?